app/views/transactions/in_review.html.haml

```diff
@@ -1,12 +1,14 @@
 -# TODO: app/views/accounts/transactions_in_review.html.haml is similar
 
+- if @order_details.blank?
+  - flash.now[:alert] = text("transactions.none_recently_reviewed")
```

Comment:
Since this is happening in the view, can we use `%p.notice=` instead of setting a flash? I also think the blue`notice` is a little calmer. And can we put it below the filters? That's closer to what we do in other places. And that'll make it clearer that the filters don't apply to recently closed.